packages/block-library/src/image/edit.native.js

```diff
@@ -153,6 +157,14 @@ export default class ImageEdit extends React.Component {
 			</Toolbar>
 		);
 
+		const inlineToolbarButtons = (
```

Comment:
We don't really need an extra variable here, we could inline it in the main markup, but I don't think it does any harm having it here...

Reply:
I was following the `toolbarEditButton` example, but I'll move it inline in the next PR.
Thank you!